build.py

```diff
@@ -102,6 +102,8 @@ def do_make(target=None, log=False):
     if silent:
         make_args += "-s "
     make_args += "ARCH=%s " %arch
+    if cc:
+        make_args += "CC=%s " %cc
```

Comment:
I recommend doing the same for `HOSTCC` for building host utilities used during the builds.